server/modules/authentication/discord/authentication.js

```diff
@@ -6,9 +6,22 @@
 
 const DiscordStrategy = require('passport-discord').Strategy
 const _ = require('lodash')
+const DiscordOauth2 = require('./node_modules/discord-oauth2/index.js') //I don't remember why I 
+
+// Checks for the existence of all of the required role IDs in the member's guild role IDs.
+function hasRoles(memberRoles, authRoles) {
+  if (authRoles.every(value => {
+    return memberRoles.includes(value)
+  })) {
+    return true
+  } else {
+    return false
+  }
+};
```

Comment:
Since we've moved the logic of the `hasRoles` method it is unnecessary right now.

```suggestion
```